app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="maxBookmarks">10</integer>
```

Comment:
@brianreinkedeveloper that's cool you found a way round an issue and you figured out how to integers.xml.why can the user only have 10 bookmarks?